### ISSUE TITLE
luci-app-wireguard: typographic error

### ISSUE THREAD

**Issue by castillofrancodamian** [openwrt/luci#5292]
It should be "hace 20s".

![1](https://user-images.githubusercontent.com/47670442/130364450-9463a643-129e-482e-b9f8-41246e9517d4.png)

**Comment by jow-** [comment]
The translation strings need to be changed in the source, using something like:
```javascript
		if (seconds < 60) {
			ago = _('%ds ago').format(seconds);
		} else if (seconds < 3600) {
			ago = _('%dm ago').format(seconds / 60);
		} else if (seconds < 86401) {
			ago = _('%dh ago').format(seconds / 3600);
```

This way translators can decide where to put the number.

An even better approach would be just doing this:

```javascript
ago = _('%t ago').format(seconds);
```

This will format the seconds value into something like `1h 12m 33s`, depending on the size of the value.

**Comment by systemcrash** [comment]
> ```js
> ago = _('%t ago').format(seconds);
> ```

Does JS appropriately localize to the correct language specific abbreviations for H:M:S?

Also, should the .po files contain `%t`, according to your suggestion?

Then I can make a drive-by PR to fix

**Comment by jow-** [comment]
> Does JS appropriately localize to the correct language specific abbreviations for H:M:S?

Not yet but I'll add that later on.

> Also, should the .po files contain %t, according to your suggestion?

Yes, they should contain

```gettext
msgid "%t ago"
msgstr "hace %t"
```

**Comment by systemcrash** [comment]
> > Does JS appropriately localize to the correct language specific abbreviations for H:M:S?
> 
> Not yet but I'll add that later on.
> 

Would prefer if we don't push things back and depend on not-yet-done parts, and stick with what exists. 

Attempt: #5319

**Comment by systemcrash** [comment]
> An even better approach would be just doing this:
> 
> ```js
> ago = _('%t ago').format(seconds);
> ```
> 
> This will format the seconds value into something like `1h 12m 33s ago`, depending on the size of the value.

The GUI currently does e.g.: Último Handshake: Tue, 31 Aug 2021 20:41:56 GMT (1m ago)

So this would present it in both absolute and relative.

**Comment by castillofrancodamian** [comment]
> This issue can be closed.

Still not solved.

![Screenshot_20211029_160442](https://user-images.githubusercontent.com/47670442/139488915-ed93d9e8-1d2e-4c07-8f9f-489abf4404e9.png)

**Comment by systemcrash** [comment]
Fixed in #5482 so it's in master.
